### DISCUSSION TITLE
Add a "Discussions" button in the Github navbar

### DISCUSSION THREAD

**Discussion by wneel** [community/community#21166] [Pull Requests]
To encourage people to invest themselves in GitHub Discussions this button would give access to a Personal Discussions panel

Like Pull requests or Issues, a way to see every Discussions we talked in, filter them by answered or not and track them in a custom Discussions feed could be very cool and more UX friendly than the actual notifications !

![image](https://user-images.githubusercontent.com/66263633/180628807-7eb2ab38-e180-4fb1-9c71-620ba7c9dd78.png)

**Comment by thuongtruong1009** [comment]
Hi @x-Wawa 
This is a simple way to open the discussion button. Follow by step by step below:
* ***Step*** ***1***: 
![image](https://user-images.githubusercontent.com/71834167/180629539-eb0d9c16-6d52-4901-8de2-e0d94c5037dd.png)
* ***Step*** ***2***:
+ Click the 1st checkbox to display the discussion button, and click to 2nd button to setup discussion origin content
![image](https://user-images.githubusercontent.com/71834167/180629573-b23a86d7-226a-47fc-a4d3-eb0854e7eaa9.png)
* ***Step*** ***3***:
+ Edit content and click the start discussion button to public 
![image](https://user-images.githubusercontent.com/71834167/180629626-2c5b3c1b-1452-4492-85b4-64b98232dbde.png)
* ***Finally***:
* Congratulate, you finish the way to display the public discussion button, which all people on GitHub can discussion about your repository
![image](https://user-images.githubusercontent.com/71834167/180629684-e3a849ba-d7c7-4055-a677-aa100f14f83a.png)

--> Follow more about me at [thuongtruong1009](https://github.com/thuongtruong1009)
